homeassistant/__main__.py

```diff
@@ -149,21 +106,18 @@ def main():
     config_path = ensure_config_path(config_dir)
 
     if args.demo_mode:
-        from homeassistant.components import frontend, demo
-
         hass = bootstrap.from_config_dict({
             frontend.DOMAIN: {},
```

Comment:
Might as well just write `frontend` and `demo` as strings. Don't know why I ever bothered using `DOMAIN`